src/item.cpp

```diff
@@ -8870,7 +8895,11 @@ item::armor_status item::damage_armor_durability( damage_unit &du, const bodypar
     // Scale chance of article taking damage based on the number of parts it covers.
     // This represents large articles being able to take more punishment
     // before becoming ineffective or being destroyed.
-    const int num_parts_covered = get_covered_body_parts().count();
+    int num_parts_covered = get_covered_body_parts().count();
+    // Acid has a higher chance of damaging equipment.
+    if( du.type->environmental && num_parts_covered > 2 ) {
+        num_parts_covered = static_cast<int>( std::max( 2.0, num_parts_covered * 0.66 ) );
+    }
```

Comment:
From the PR body:

> When an attack attempts to damage armor durability, its chances to do so are reduced according to the number of body parts the armor covers. Acid mitigates this reduction by 33% (bringing the chance for mitigation to a minimum of 1 in 2), as it's a liquid that spreads over the surface rather than a bullet or a knife trying to drill or slice a hole. Note that the resulting chance to damage durability is still very low unless the damage is obnoxiously high.

I've added the following comment:

    // Acid spreads out to cover the item, reducing the mitigation items receive from body parts covered.
    
The intent behind acid is that it damages unprotected equipment, as that would logically be a major concern. The bp coverage mitigation can make it many times harder for an item to become damaged. In testing, this mitigation was so effective that it was actually more important than more obvious factors like breathability. Without something to counter it, the best meta for avoiding acid damage would be to simply wear larger single pieces of armor.

0.66 was the number that gave me the results I wanted in testing. Removing the mitigation entirely was too much, doing nothing was too little. Making it scale off of liquid_remaining might be logical, but then we'd be double and triple dipping with the damage amount and damage/effect chances, which would have cascading effects at lower and higher volume attacks.

Reply:
Thinking more on it, what I can do is just ditch the 0.66 magic number and have acid bypass the num_parts_covered protection altogether. Then I could lower the volume of acid used in the acid_barf spell if it's damaging items too easily after that.